ml-agents/mlagents/trainers/subprocess_env_manager.py

```diff
@@ -64,6 +68,9 @@ def send(self, name: str, payload: Any = None) -> None:
     def recv(self) -> EnvironmentResponse:
         try:
             response: EnvironmentResponse = self.conn.recv()
+            if response.name == "env_close":
```

Comment:
one suggestion: create a const variable for "env_close" as it's used in 2 places.

Reply:
or even better, an enum

Reply:
Made `EnvironmentCommand` an enum with all of the different command types.  This required renaming the previous `EnvironmentCommand` class to `EnvironmentRequest`.  Changed references throughout accordingly 👍

Reply:
🎉 nice.